Commons/src/main/java/tw/waterball/judgegirl/commons/utils/functional/Otherwise.java

```diff
@@ -0,0 +1,28 @@
+package tw.waterball.judgegirl.commons.utils.functional;
+
+import java.util.function.Consumer;
+
+/**
+ * @author - johnny850807@gmail.com (Waterball)
+ */
+public class Otherwise<T> {
+    private final T t;
+
+    private Otherwise(T t) {
+        this.t = t;
+    }
+
+    public static <T> Otherwise<T> of(T t) {
+        return new Otherwise<>(t);
+    }
+
+    public static <T> Otherwise<T> empty() {
+        return new Otherwise<>(null);
+    }
+
+    public void otherwise(Consumer<T> consumer) {
+        if (t != null) {
+            consumer.accept(this.t);
+        }
+    }
+}
```

Comment:
A semantic sugar that can be used similar with `Optional<T>`

So you can write something like
```java
doSomethingIfXXX(...)
    .otherwise(() -> ...)
```